### PR TITLE
Pull Request for sam1123

### PR DESCRIPTION
**Pull request by sammcdsam**
Pull Request for the assignment intro to github. 
Included are my files from the intro to sphinx assignment as well as my partners code.

### CODE REVIEW

**Comment by bjones1** [comment]
Delete all the files in `_build` and rebase/merge with the latest commits in master.

File: sam1123/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# ignore codechat_config.yaml and conf.py
```
Comment (bjones1) [inline sam1123/.gitignore]: Good idea, but not needed here -- delete this file.

File: sam1123/_build/_sources/index.rs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (bjones1) [inline sam1123/_build/_sources/index.rs]: Delete this file.

File: sam1123/feedback.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+********
```
Comment (bjones1) [inline sam1123/feedback.rst]: Delete this file.

File: sam1123/index.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
+.. template documentation master file, created by
```
Comment (bjones1) [inline sam1123/index.rst:1]: Delete this file.

File: sam1123/shadows_of_knight_ep1.py [hunk 5]
```diff
@@ -0,0 +1,97 @@
+# ***************************
```
Comment (bjones1) [inline sam1123/shadows_of_knight_ep1.py]: This is your partner's code, correct? If so, delete it.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
Sam, this is a good start. Respond to my comment by midnight on 3-Mar.

**Comment by bjones1** [comment]
I haven't seen any progress on this...

**Comment by sammcdsam** [comment]
Sorry Dr. Jones I didnt realize you wanted me to reply here on Thursday. I was working on a presentation on a different class and thought I could just squeeze this in before the morning.

File: sam1123/sam1123_codechat_intro.py [hunk 6]
```diff
@@ -0,0 +1,141 @@
+# *****************************************************
+# Coding with Reflection - Death-First Search (sam1123)
+# *****************************************************
+# Solving the the `death-first-search puzzle <https://www.codingame.com/ide/puzzle/death-first-search-episode-1>`_ from codingame
+#
```
Comment (bjones1) [inline sam1123/sam1123_codechat_intro.py:5]: Remove notes and integrate their suggestions into your code.

File: sam1123/sam1123_codechat_intro.py [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+# My actual solution just ended up being to randomly cut a path in the graph. If the virus was next to an exit the program then cuts that exit path, and the process repeats. **This solution passed all the test cases of the codingame.** 
+
+# =====================
+# Imports and Variables
+# =====================
+# 
+# This section imports libraries and initalizes variables used in the solution below. 
+#
+# .. note::
```
Comment (bjones1) [inline sam1123/sam1123_codechat_intro.py]: Remove / integrate.

File: sam1123/sam1123_codechat_intro.py [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+d = {}
+
+
+# ================
+# Read in the data
+# ================
+# Auto-generated code below aims at helping you parse
+# the standard input according to the problem statement.
+# 
+# n: the total number of nodes in the level, including the gateways
```
Comment (bjones1) [inline sam1123/sam1123_codechat_intro.py]: Use a bulleted list of a definition list:

```
# :l: the number of links
# :e: the number of exit gateways
```

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
It looks like a lot of the notes I left in the previous assignment haven't been addressed -- make sure to improve your code based on that feedback, then I can provide more review.

**Comment by bjones1** [comment]
Still waiting on these changes...

**Comment by bjones1** [comment]
It looks like there's one more section of notes you haven't edited combined, but everything else looks fine. See https://github.com/LiterateProgramming/exploring-github/pull/11/files#diff-afd97f031b3bbad2bf8c611611d502d73d47d0d960cf088c272dbea5df432f91R6.

**Comment by bjones1** [comment]
Ping...still waiting on this...

**Comment by bjones1** [comment]
I see some progress, but still unresolved problems. Comment here when your code is ready for review.

**Comment by bjones1** [comment]
Or message me for more details on what I'd like to see.

**Pull request merged**